### DISCUSSION TITLE
Возможна ли пакетная загрузка статей

### DISCUSSION THREAD

**Discussion by GPolyakov** [maxsite/cms#465] [Q&A] [answered]
Добрый день.

Подскажите пожалуйста, есть ли возможность пакетной загрузки статей? 
Зачем это вообще - заказываю статьи на биржах, и хочется понимать как их пачкой грузить. Пока видится только через SQL напрямую можно.
Может есть плагин какойнить, который как минимум может по RSS загружать или плагин который из архива может файлы обработать, и завести статьи

**Comment by vimruler** [comment, marked as answer]
А чем не подходит стандартное средство AutoPost, ссылка на которое есть в заголовке формы создания новой записи или по ссылке /admin/auto_post ?

Там можно подсовывать текстовые файлы и заливать записи очень просто и удобно.

[ОФФТОП] Хорошо бы ещё было средство экспорта записей, чтобы можно было удобно перекидывать записи между сайтами :-)

**Reply by garuda-dp** [reply]
Спасибо большое, посмотрю на auto_post,  ток разбираюсь с CMS

**Comment by maxsite** [comment]
> [ОФФТОП] Хорошо бы ещё было средство экспорта записей, чтобы можно было удобно перекидывать записи между сайтами :-)

Он есть, правда для одной записи. Кнопка возле выпадающего списка записей.

**Comment by nvda-ru** [comment]
Для импорта из RSS можно сказать, был интересный плагин
http://maxhub.ru/page/plugin-q-rss-import

Конечно, у него не хватало настройки в какую рубрику публиковать из какой ленты. Какой статус задавать записям: public, draft, privat. 

Несколько лет назад, пробовал это плагин из интереса, но он уже тогда добавлял запись, но у каждой записи не проставлялась дата или статус записи.
В общем, при переходе по ссылке, самой записи не было.
